test/mpi_tests/regridder_mpi_tests.jl

```diff
@@ -16,7 +16,7 @@ REGRID_DIR = @isdefined(REGRID_DIR) ? REGRID_DIR : joinpath("", "regrid_tmp/")
 # Set up MPI communications context
 # Note that runs will hang if a context is initialized twice in the same file,
 # so this context should be shared among all tests in this file.
-comms_ctx = ClimaComms.SingletonCommsContext(ClimaComms.CPUDevice())
+comms_ctx = ClimaComms.SingletonCommsContext(ClimaComms.CPUSingleThreaded())
```

Comment:
```suggestion
comms_ctx = ClimaComms.SingletonCommsContext(ClimaComms.device())
```
This way this file will work with unthreaded, threaded, or gpu